saleor/graphql/order/utils.py

```diff
@@ -27,8 +31,36 @@ def validate_total_quantity(order: "Order", errors: T_ERRORS):
         )
 
 
-def validate_shipping_method(order: "Order", errors: T_ERRORS):
-    error = None
+def get_shipping_method_availability_error(
+    order: "Order",
+    method: Optional["ShippingMethodData"],
+    manager: "PluginsManager",
+):
```

Comment:
It's really hard to read. How about this version:
```
is_valid = False
if method:
    valid_methods_ids = get_valid_methods_ids(order, manager)
    is_valid = method.id in valid_methods_ids
```

Reply:
We still need to check the `active` flag. I'd keep this one as it is unless someone else complains about this. Extracting this to a function that's going to have only one occurrence doesn't seem to be worth it